tasks/keepalived_install.yml

```diff
@@ -45,9 +45,9 @@
   tags:
     - keepalived-repo
 
-- name: Ensure python apt is installed
+- name: Ensure python-apt or python3-apt is installed
   apt:
```

Comment:
maybe, for clarity, we could do a variable, `packagename:` whose content can be
`{{ ansible_python['version']['major'] == 3 | ternary('python3-apt','python-apt') }}`, or something similar?